docs/how-tos/deploy-ml.md

```diff
@@ -137,34 +289,39 @@ Click **Train model** and follow the prompts.
 You can choose between
 
 - training a new model or updating a model
-- You can use a **Built-in training** script or a [custom training script](/services/ml/training-scripts/).
+- You can use a **Built-in training** script or a [training script](/services/ml/training-scripts/).
 
 Click **Next steps**.
 
+{{<imgproc src="/tutorials/data-management/shapes-dataset.png" resize="1200x" declaredimensions=true style="max-width:500px" alt="The shapes dataset." class="imgzoom fill aligncenter">}}
+
 {{% /tablestep %}}
 {{% tablestep %}}
 **2. Select the details for your ML model**
 
 1. Enter a name or use the suggested name for your new model.
```

Comment:
[out-of-diff opt] I don't love the look of numbered steps inside a numbered step. Consider just splitting into paragraphs within this step.